dev/META6.json

```diff
@@ -12,11 +12,13 @@
     "Cro::HTTP",
     "Cro::HTTP::Session::Pg",
     "Cro::OpenAPI::RoutesFromDefinition",
+    "Cro::Uri",
```

Comment:
It's better to name distributions here rather than modules; this should probably be `Cro::Core`.

Reply:
Done.